tests/api_test.py

```diff
@@ -2269,6 +2269,21 @@ def foo_jvp(primals, tangents):
         ),
         lambda: api.jvp(f, (2.,), (1.,)))
 
+  def test_multiple_rule_invocations(self):
+    from jax.scipy.special import expit
```

Comment:
This tests is only effective because of the way we define custom_jvp for expit. If that aspect changes we loose the benefit of this test. Wouldn't it make sense to copy over here the custom_jvp for expit?

Reply:
Good idea!

Reply:
Done!